pkg/pipeline/PipelineBuilder.go

```diff
@@ -1413,6 +1419,20 @@ func (impl PipelineBuilderImpl) PatchCiPipeline(request *bean.CiPatchRequest) (c
 	ciConfig.AppWorkflowId = request.AppWorkflowId
 	ciConfig.UserId = request.UserId
 	if request.CiPipeline != nil {
+		client, err := impl.K8sUtil.GetClientForInCluster()
+		if err != nil {
+			impl.logger.Errorw("exception while getting unique client id", "error", err)
+			return nil, err
+		}
+		cm, err := impl.K8sUtil.GetConfigMap(ConfigMapNamespace, DashboardConfigMap, client)
+		datamap := cm.Data
```

Comment:
What if cm is nil, handle that case also

Reply:
done

Reply:
not able to see it handled

Reply:
might not be showing that time, handled it.